ui/themes/app.js

```diff
@@ -614,10 +614,18 @@ export const styles = (theme) => ({
     flex: 1,
     display: 'flex',
     flexDirection: 'column',
+    [theme.breakpoints.up('sm')]: {
+      width: `60%`,
+      overflowX: 'clip',
```

Comment:
@dottharun, confirm that no other page is negatively affected by these broad-sweeping changes, please.

Reply:
i didn't understand @dottharun , there is issue when using this breakpoint on other page right?

Reply:
Yes.
- Now resolved 2 side-effects caused on other pages.
- Nothing more I could find caused by this PR.